dpnp/dparray.pyx

```diff
@@ -113,17 +193,27 @@ cdef class dparray:
             self._dparray_strides.push_back(strides_it)
 
         # data
-        if memptr is None:
+        if memory is None:
             self._dparray_data = dpnp_memory_alloc_c(self.nbytes)
+            self.queue = c_dpctl.get_current_queue()
```

Comment:
DPNP localized all queue related things inside the backend. Python and Cython layers has no relations to SYCL queue

Reply:
move this to backend or use existed backend interface

Reply:
Yes, it does. When `dpctl` changes the default queue, the `sycl::get_pointer_type( _dparray_data, DPNP_CURRENT_QUEUE().get_context())`  may return `"uknown"`, having the `queue` onto which the memory was allocated at least would allow `dpnp` to access the data.